virtwho/main.py

```diff
@@ -151,6 +144,16 @@ def main():
         logger.error(str(e))
         exit(1, "virt-who can't be started: %s" % str(e))
 
+    lock = PIDLock(PIDFILE)
+    if lock.is_locked()\
+            and not executor.options[VW_GLOBAL]['oneshot'] \
+            and not executor.options[VW_GLOBAL]['status'] \
+            and not executor.options[VW_GLOBAL]['print']:
```

Comment:
Maybe I would change order of checks and I would add `lock.is_locked()` to the end, because it is really not necessary to try to read lock file and check if the process is still running, when you use e.g. `--status` option.